xilinx/xc7/archs/artix7/tiles/tieoff/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+project_ray_tile(
+  ARCH artix7
+  TILE TIEOFF
+  SITE_AS_TILE
+  SITE_TYPES TIEOFF/TIEOFF
+  EQUIVALENT_SITES TIEOFF
+  )
```

Comment:
This for sure needs to be removed, there is no need for modeling TIEOFFs as tiles. In VPR we do not model them as sites and their outputs are connected to the synthetic global constant network.